airbyte-config/config-models/src/main/resources/types/SuggestedStreams.yaml

```diff
@@ -0,0 +1,14 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SuggestedStreams.yaml
+title: SuggestedStreams
+description: A source's suggested streams.  These will be suggested by default for new connections using this source.  Otherwise, all streams will be selected.
+type: object
+required:
+additionalProperties: true
+properties:
+  streams:
+    type: array
+    description: An array of streams that this connector suggests the average user will want.  SuggestedStreams not being present for the source means that all streams are suggested.  An empty list here means that no streams are suggested.
```

Comment:
Thanks for thinking this through and laying it out, I'm sure it'll be helpful in some weird issue a year from now